docs/contributing/get-started.md

```diff
@@ -50,66 +50,4 @@ Let the project team know in the #contributors channel on Slack.
-| [connector-sdk](https://github.com/openfaas-incubator/connector-sdk)         | Build your own event connectors for OpenFaaS |
-| [vcenter-connector](https://github.com/openfaas-incubator/vcenter-connector) | Trigger OpenFaaS Functions from events in VMware vCenter |
-| [faas-federation](https://github.com/openfaas-incubator/faas-federation) | Federate two OpenFaaS installations into one API |
-| [faas-memory](https://github.com/openfaas-incubator/faas-memory) | An OpenFaaS Provider example using memory for state. |
-
-##### Training & tutorials
-
-| Repository        | Headline                         |
-|:------------------|:---------------------------------|
-| [workshop-vscode](https://github.com/openfaas-incubator/workshop-vscode) | Run a Kubernetes workshop with VSCode in the browser |
-| [openfaas-linkerd2](https://github.com/openfaas-incubator/openfaas-linkerd2) | Lightweight Serverless on Kubernetes with mTLS and traffic-splitting with Linkerd2 |
-| [openfaas-function-auth](https://github.com/openfaas-incubator/openfaas-function-auth) | Examples of authentication in OpenFaaS Serverless functions. |
-
-##### Templates
-
-| Repository        | Headline                         |
-|:------------------|:---------------------------------|
-| [node10-express-template](https://github.com/openfaas-incubator/node10-express-template) | Node.js 10 Express template providing additional context and control over the HTTP response from your function |
-| [node10-express-service](https://github.com/openfaas-incubator/node10-express-service) | Template with Node.js and Express.js exposed fully |
-| [node8-express-template](https://github.com/openfaas-incubator/node8-express-template) | Node.js 8 template for OpenFaaS with HTTP via Express.js |
-| [golang-http-template](https://github.com/openfaas-incubator/golang-http-template) | Golang template providing additional control over the HTTP request and response.|
-| [powershell-http](https://github.com/openfaas-incubator/powershell-http-template) | PowerShell HTTP template |
-| [ruby-http](https://github.com/openfaas-incubator/ruby-http) | A Ruby HTTP template for OpenFaaS |
-| [python-flask-template](https://github.com/openfaas-incubator/python-flask-template) | OpenFaaS templates for Python 2.7/3.6 with Flask |
-| [python3-debian](https://github.com/openfaas-incubator/python3-debian) | Template for Python3 on Debian for data-science / compiled pip modules |
-| [perl-template](https://github.com/openfaas-incubator/perl-template) | Perl template for OpenFaaS |
+All source is organised under two repositories on GitHub: openfaas and openfaas-incubator. Some prototyping may take place under the personal accounts of the [Members Team](https://github.com/orgs/openfaas/people).  Please refer to the [Main Repositories](./contributing/main-repositories.md/) page for full details.
```

Comment:
Is `/main-repositories.md/` correct? I haven't seen that syntax

Reply:
i think the dot is wrong